TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -373,7 +379,7 @@ inline void advance() {
  ap_uint<4> stubmask_;
  ap_uint<1> second_;
  ap_uint<1> phiPlus_;
- ap_int<2> shift_;
+ ap_uint<1> phiProjBin_;
```

Comment:
@jasonfan393 I think we could eliminate a magic number by taking the type from `ProjectionRouterBuffer`:
```suggestion
 ProjectionRouterBuffer<BARREL, AllProjectionType>::PHIPROJBIN phiProjBin_;
```

And this might apply to other types within this class…